node/src/index.js

```diff
@@ -1,185 +1,203 @@
-const _ = require('lodash')
-const path = require('path')
-const express = require('express')
-const hubspot = require('@hubspot/api-client')
-const bodyParser = require('body-parser')
-require('./config')
-
-const PORT = 3000
-const OBJECTS_LIMIT = 30
-const CLIENT_ID = process.env.HUBSPOT_CLIENT_ID
-const CLIENT_SECRET = process.env.HUBSPOT_CLIENT_SECRET
-const SCOPES = 'contacts'
-const REDIRECT_URI = `http://localhost:${PORT}/oauth-callback`
+const _ = require("lodash");
```

Comment:
To be consistent I'd copy main prettier rules from https://git.hubteam.com/HubSpot/prettier-config-hubspot/blob/master/index.js#L7.

It uses single quote instead of double as default.

Reply:
added the config @atanasiuk-hubspot provided, rerun prettier.